lib/_stream_readable.js

```diff
@@ -538,12 +538,12 @@ Readable.prototype.pipe = function(dest, pipeOpts) {
   }
   // This is a brutally ugly hack to make sure that our error handler
   // is attached before any userland ones.  NEVER DO THIS.
-  if (!dest._events || !dest._events.error)
+  if (!dest._events || !dest._events.has('error'))
     dest.on('error', onerror);
-  else if (Array.isArray(dest._events.error))
-    dest._events.error.unshift(onerror);
+  else if (Array.isArray(dest._events.get('error')))
+    dest._events.get('error').unshift(onerror);
   else
-    dest._events.error = [onerror, dest._events.error];
+    dest._events.set('error', [onerror, dest._events.get('error')]);
```

Comment:
If this gets merged, will 1.0.* and 1.1.* branches of readable-stream receive pathches?
If yes, there are not so many packages that specify a fixed version of readable-stream as a dependency.